packages/io/amf-serializer/index.js

```diff
@@ -8,104 +8,103 @@ Copyright (c) 2018 JSCAD Organization https://github.com/jscad
 All code released under MIT license
 
 Notes:
-1) CAG conversion to:
+1) geom2 conversion to:
```

Comment:
I like these in all the serializers !
Perhaps it would make sense to add this info to the readme instead of the index ? or both places ?

Reply:
I believe all README files have these details. :)

Reply:
not in such details though right ?